### DISCUSSION TITLE
ISO to NVMe

### DISCUSSION THREAD

**Discussion by cyberpika2** [Security-Onion-Solutions/securityonion#2663] [Unsupported Versions] [answered]
Has anyone been able to install the ISO to NVMe SSD?

**Comment by weslambert** [comment]
Have you tried it yet?  I can't think of anything that would be stopping you at the moment -- we 've used NVMe before for at least the /nsm portion.

**Reply by cyberpika2** [reply]
Yes, and no luck. 
I haven't been able to get through the ISO installer on bare metal or in VMware using the nvme driver. Other controllers work. 
Hangs up before the partitioning dialog.
Verified ISO signatures. 2.3.21
Related to Issue 1480 but affects bare metal installs as well.

**Comment by TOoSmOotH** [comment, marked as answer]
I just pushed a change to the kickstart script to try and address this. Should see something in 2.3.30 to address this.
